Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,31 @@
 import React from 'react';
 import './App.css';
+import NumberButton from './components/ButtonComponents/NumberButton';
+// import ActionButton from './components/ButtonComponents/ActionButton';
+import CalculatorDisplay from './components/ButtonComponents/CalculatorDisplay';
```

Comment:
`CalculatorDisplay` is under the `DisplayComponents` folder.
`import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';`